src/app/shared/components/cluster-summary/template.html

```diff
@@ -607,6 +627,38 @@
               </km-property>
             </ng-container>
 
+            <!-- Nutanix Nodes -->
+            <ng-container *ngIf="cluster.spec.cloud?.nutanix">
+              <km-property>
+                <div key>Image Name</div>
+                <div value>{{machineDeployment.spec.template.cloud.nutanix.imageName}}</div>
+              </km-property>
+              <km-property>
+                <div key>Subnet Name</div>
+                <div value>{{machineDeployment.spec.template.cloud.nutanix.subnetName}}</div>
+              </km-property>
+              <km-property>
+                <div key>CPUs</div>
+                <div value>{{machineDeployment.spec.template.cloud.nutanix.cpus}}</div>
+              </km-property>
+              <km-property>
+                <div key>Cores per CPU</div>
+                <div value>{{machineDeployment.spec.template.cloud.nutanix.cpuCores}}</div>
+              </km-property>
+              <km-property-boolean *ngIf="machineDeployment.spec.template.cloud.nutanix.cpuPassthrough"
```

Comment:
Can we move this boolean property to the end? I was trying to keep them at the end not to break the standard properties flow.